source/style-guide/writing/use-effective-verbs.txt

```diff
@@ -336,11 +336,11 @@ Be careful not to ascribe human feelings, motivations, and actions to
 inanimate objects. For example, a web application doesn't *know*,
 *need*, *remember*, *see*, *think*, *understand*, or *want*.
 
-Be careful not to ascribe human qualities, traits, or emotions to
-inanimate objects like software, hardware, or interfaces. Often, we use
-anthropomorphic words in an attempt to make difficult topics more understandable or relatable. This can actually make the
-concept harder to grasp for ESL readers or those less familiar with
-metaphors.
+Often, we use anthropomorphic words to make difficult topics more
```

Comment:
**suggestion:** The use of *anthropomorphic* was why I added "humanize" first so people could understand that the word means the same. Consider adding that back in.

Also, do they have to be relatable or understandable? Could they be one or the other?